src/app/Scenes/HomeView/Sections/HomeViewSectionArtworks.tsx

```diff
@@ -54,41 +52,44 @@ export const HomeViewSectionArtworks: React.FC<HomeViewSectionArtworksProps> = (
     }
   }
 
+  const onSectionViewAll = () => {
+    if (viewAll?.href) {
```

Comment:
you can also use `getSectionHref` helper here
```suggestion
    const sectionHref = getSectionHref(...)
```

Reply:
As written, the `getSectionHref` helper was difficult to use as I needed to know the specific state of the viewAll trigger in order to customize the analytics payload. Might be worth another pass to see if this can be better DRY'd up between section components but I opted for a more explicit approach in this PR.

Reply:
oki - maybe we can later update it to return the state `default/custom` etc and the `href`. It's alright since that's not a blocker here but nice to have

Reply:
Yeah, another tricky element is that often the component itself knows best how to resolve a "default" or unspecified viewAll implementation.

For HomeViewSectionActivity, that might mean explicitly navigating to `/notifications` but for HomeViewSectionArtworks that might mean navigating to `/home-view/sections/:id`. I don't think it's guaranteed that all section types implemented within Eigen will support a generic viewAll behavior, so it seems very component-specific at the moment.